Packages/Tracking/Physical Hands/Runtime/Scripts/UI/PhysicalHandsButtonHelper.cs

```diff
@@ -0,0 +1,57 @@
+/******************************************************************************
+ * Copyright (C) Ultraleap, Inc. 2011-2024.                                   *
+ *                                                                            *
+ * Use subject to the terms of the Apache License 2.0 available at            *
+ * http://www.apache.org/licenses/LICENSE-2.0, or another agreement           *
+ * between Ultraleap and you, your company or other organization.             *
+ ******************************************************************************/
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace Leap.Unity.PhysicalHands
+{
+    public class PhysicalHandsButtonHelper : MonoBehaviour, IPhysicalHandContact, IPhysicalHandHover
```

Comment:
Should this live in the Utils folder?

Reply:
maybe but seems weird to put it alongside all the static utilities as it is added to a GO at runtime